components/admin_console/database/index.tsx

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {connect} from 'react-redux';
+
+import MigrationsTable from './migrations_table';
+
+export default connect()(MigrationsTable);
```

Comment:
We'll end up needing this for what I mentioned, but if we weren't connecting this to redux, then this file could just re-export `MigrationsTable` without passing it through `connect`.